portfolio/src/main/java/com/google/sps/entities/Comment.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.entities;
+
+import lombok.*;
```

Comment:
Would it be better to specify the specific components of lombok that we are importing?

Reply:
I think so, I'll put it in to be explicit.